scripts/safe_template_linter.py

```diff
@@ -1511,7 +1664,7 @@ def _check_filters(self, expression, context, has_page_default, results):
 
         """
         # finds "| n, h}" when given "${x | n, h}"
-        filters_regex = re.compile(r'\|[a-zA-Z_,\s]*\}')
+        filters_regex = re.compile(r'\|[a-zA-Z0-9_.,\s]*\}')
```

Comment:
`\w`:

... matches any alphanumeric character and the underscore; this is equivalent to the set [a-zA-Z0-9_].
https://docs.python.org/2/library/re.html